boards.txt

```diff
@@ -14375,3 +14375,153 @@ AirM2M_CORE_ESP32C3.menu.DebugLevel.verbose=Verbose
 AirM2M_CORE_ESP32C3.menu.DebugLevel.verbose.build.code_debug=5
 
 #############################################################
+
+
+SEEED_XIAO_WIFI.name=SEEED_XIAO_WIFI
+SEEED_XIAO_WIFI.vid.0=0x2886
+SEEED_XIAO_WIFI.pid.0=0x0047
+
+SEEED_XIAO_WIFI.bootloader.tool=esptool_py
+SEEED_XIAO_WIFI.bootloader.tool.default=esptool_py
+
+SEEED_XIAO_WIFI.upload.tool=esptool_py
+SEEED_XIAO_WIFI.upload.tool.default=esptool_py
+SEEED_XIAO_WIFI.upload.tool.network=esp_ota
+
+SEEED_XIAO_WIFI.upload.maximum_size=1310720
+SEEED_XIAO_WIFI.upload.maximum_data_size=327680
+SEEED_XIAO_WIFI.upload.flags=
+SEEED_XIAO_WIFI.upload.extra_flags=
+SEEED_XIAO_WIFI.upload.use_1200bps_touch=false
+SEEED_XIAO_WIFI.upload.wait_for_upload_port=false
+
+SEEED_XIAO_WIFI.serial.disableDTR=false
+SEEED_XIAO_WIFI.serial.disableRTS=false
+
+SEEED_XIAO_WIFI.build.tarch=riscv32
+SEEED_XIAO_WIFI.build.target=esp
+SEEED_XIAO_WIFI.build.mcu=esp32c3
+SEEED_XIAO_WIFI.build.core=esp32
+SEEED_XIAO_WIFI.build.variant=SEEED_XIAO_WIFI
+SEEED_XIAO_WIFI.build.board=SEEED_XIAO_WIFI
+SEEED_XIAO_WIFI.build.bootloader_addr=0x0
+
+SEEED_XIAO_WIFI.build.cdc_on_boot=0
```

Comment:
Your board seems to be using the native USB port for flashing and communication. Are you sure that you do not want this option to be on by default? If you decide to do so, you also need to switch the order of the menu items for CDCOnBoot, because the first option is taken by default

Reply:
![image](https://user-images.githubusercontent.com/47217650/167233458-ab977be7-998c-4079-ba00-f67acca1faee.png)
Do you mean like here set to 1?      but I saw other ESP32 C3 boards that all use this parameter.

Reply:
done!   I changed it.

Reply:
You should also set it to 1 in the line above: `SEEED_XIAO_WIFI.build.cdc_on_boot=1`